pkg/controller/audit/actuator.go

```diff
@@ -560,7 +561,7 @@ func seedObjects(auditConfig *v1alpha1.AuditConfig, secrets map[string]*corev1.S
 				},
 			},
 			Spec: policyv1.PodDisruptionBudgetSpec{
-				MinAvailable: utils.IntStrPtrFromInt32(1),
+				MinAvailable: ptr.To(intstr.FromInt32(1)),
```

Comment:
Historically, we have an own pointer package. :)

```suggestion
				MinAvailable: pointer.Pointer(intstr.FromInt32(1)),
```